packages/tools/u-boot/package.mk

```diff
@@ -35,18 +35,24 @@ esac
 
 make_target() {
   if [ -z "$UBOOT_SYSTEM" ]; then
```

Comment:
What if UBOOT_SYSTEM is defined? That will be true for building single images for Allwinner and Rockchip projects. In that case tools will never be build, right?

Reply:
Will be, please look at addition at line 44.

Reply:
ok, sorry for the noise